recipes/rnaquast/meta.yaml

```diff
@@ -32,6 +32,7 @@ requirements:
 test:
   commands:
     - rnaQUAST.py --test
```

Comment:
Wondering why the first test passed before. According to the test logs it seems that BLAST runs during the triggered test.

Reply:
If you check the test output of https://github.com/bioconda/bioconda-recipes/pull/23025 you find: 

```
15:36:01 BIOCONDA INFO (ERR) [Jul  6 15:36:01] SOUT Getting blast database for /usr/local/bin/rnaQUAST_test_output/tmp/Saccharomyces_cerevisiae.R64-1-1.75.isoforms.fa
15:36:01 BIOCONDA INFO (ERR) [Jul  6 15:36:01] SERR makeblastdb: error while loading shared libraries: libbz2.so.1: cannot open shared object file: No such file or directory
15:36:01 BIOCONDA INFO (ERR) [Jul  6 15:36:01] SERR ERROR! makeblastdb failed!
```

Reply:
This time this seems absent..

Reply:
I don't get the details. The test seems to be executed twice. And it failed before in the part after the output:

```
14:43:05 BIOCONDA INFO TEST START via mulled-build recipes/rnaquast
```